.github/workflows/lint-and-format.yml

```diff
@@ -1,6 +1,6 @@
 name: Lint and Format
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
This is actually safer to use when dealing with forked repos apparently https://dev.to/suzukishunsuke/secure-github-actions-by-pullrequesttarget-641